src/tutorials/lua_basics_lightbulb.md

```diff
@@ -770,205 +624,162 @@ There are two ways to change a trigger's label, by going to the trigger's **Prop
 
```

Comment:
This seems like the start of a new phase, so separate it out with an h2 (`##`), Changing the Interaction Label (or something that sounds clearer than that about what we're actually doing, and lead with the step described in the next comment.